__tests__/app.test.js

```diff
@@ -21,3 +27,28 @@ describe("CORE: GET /api/topics", () => {
       });
   });
 });
+
+describe("CORE: GET /api/articles", () => {
+  it("retrieves array of articles in decending order by date", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        console.log("article 🙃", articles);
+
+        articles.forEach((article) => {
```

Comment:
If you're going to use a forEach we also need to assert that there is a length so that this does not pass as a false positive.